sharktank/sharktank/models/deepseek/toy_deepseek.py

```diff
@@ -49,9 +48,7 @@ def generate(seed):
             v_head_dim=32,
             expert_count=expert_count,
             expert_used_count=used_experts,
-            model_arch="deepseek",
-            expert_score_func="sigmoid",
-            route_scale=2.0,
+            model_arch="deepseek2",
```

Comment:
I know its dumb but could we do `deepseek_v2`. I think thats how they do their naming scheme.